### PR TITLE
BZ 2015486 ACM24: Bump ai-ui-lib to v1.5.39-3-acm24

### PR DESCRIPTION
**Pull request by mareklibra**
BZ 2015486: Add warning about no available agents in Hosts selection step

Contains:
- Backport #940 
- https://github.com/openshift-assisted/assisted-ui-lib/releases/tag/v1.5.39-3-acm2.4

### CODE REVIEW

**Comment by jelkosz** [comment]
/hold 
needs a little more verification before merge

**Review by jeswanke — APPROVED** [review]

**Pull request merged**
